core/src/main/resources/schema/app/20230110074614_DISTX-739_Fix_EnableLoadBalancer_set_to_false_irrespective_of_true_in_custom_cluster_definition.sql

```diff
@@ -0,0 +1,9 @@
+-- // DISTX-739 Fix EnableLoadBalancer set to false irrespective of true in custom cluster definition
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE clustertemplate ADD COLUMN IF NOT EXISTS enableloadbalancer BOOLEAN;
```

Comment:
dont we need explicit default value for this field?

Reply:
I have set the default to true